docs/source/developer_resources.rst

```diff
@@ -31,6 +31,44 @@ To run flake locally call:
 
     tox -e flake8
 
+Python Type Hints
```

Comment:
Looking for feedback on this. Specifically:
- If you don't use VSCode, what do you use? we should check if there's a suitable extension for it
- Is the usage of types clear and sufficient?

Reply:
- I use VSCode
- I think I'm following. Typing is not mandatory but prefered going forward. If I run into issues I can always use `# type ignore`

Reply:
I use Sublime and it looks like there's a plugin already that ties mypy in: https://github.com/fredcallaway/SublimeLinter-contrib-mypy